### ISSUE TITLE
Option to add ${EMPTY} with NormalizeAssignments

### ISSUE THREAD

**Issue by jyrlindr** [MarketSquare/robotframework-tidy#474]
It would be nice if Robotidy could add ${EMPTY} to robot assignments if configured so.
This could be done using `NormalizeAssignments` or there could a specific transformer for this.

At the moment, we have started to use following configuration:
```
NormalizeAssignments:equal_sign_type=space_and_equal_sign:equal_sign_type_variables=space_and_equal_sign
```
We have quite a few lines where lines like:
```
*** Variables ***
${MENU_NODE}
${MENU_NODE_UI_NAME}
```
become
```
*** Variables ***
${MENU_NODE} =
${MENU_NODE_UI_NAME} =
```
This look a bit funny though.

We are also using Robocop rule: `empty-variable` (as ERROR)

It would be nice if Robotidy could automatically fix these for us

**Comment by bhirsz** [comment]
Uh, right. But I'm wondering if we should add it to NormalizeAssignments or rather add whole new transformer? like 'ReplaceEmptyValues'. It would take care of any other ambigious case like this:

@{MENU_OPTIONS}    to  @{EMPTY}

etc

**Comment by bhirsz** [comment]
Added ``ReplaceEmptyValues`` transformer which is part of the just released Robotidy 4.0. However I recommend using Robotidy 4.0.1 when it is (soon) released since I also introduced regression.
